ignite/pkg/xtime/clock.go

```diff
@@ -0,0 +1,49 @@
+package xtime
+
+import "time"
+
+// Clock represents a clock that can retrieve current time
+type Clock interface {
+	Now() time.Time
+	Add(duration time.Duration)
```

Comment:
Unless you have future usage in mind, this method is never used, even in the tests